deltachat-ios/Chat/Views/Cells/ImageTextCell.swift

```diff
@@ -44,7 +44,7 @@ class ImageTextCell: BaseMessageCell {
 
     override func update(dcContext: DcContext, msg: DcMsg, messageStyle: UIRectCorner, showAvatar: Bool, showName: Bool) {
         messageLabel.text = msg.text
-        bottomCompactView = msg.type != DC_MSG_STICKER && msg.text?.isEmpty ?? true
+        bottomCompactView = msg.type != DC_MSG_STICKER && !msg.hasHtml && msg.text?.isEmpty ?? true
```

Comment:
in other words, the "Full View" is shown when the type is a sticker or some text is set?

i find that hard to read :)

Reply:
You're right about both. I added a second commit that improves the readability.